clients/src/test/java/org/apache/kafka/common/requests/OffsetFetchRequestTest.java

```diff
@@ -42,17 +43,21 @@ public void testWithMultipleGroups(short version) {
                     .setTopics(List.of(
                         new OffsetFetchRequestData.OffsetFetchRequestTopics()
                             .setName("foo")
+                            .setTopicId(Uuid.randomUuid())
+                            .setTopicId(Uuid.randomUuid())
```

Comment:
Is there an accidental line duplication here?

Reply:
There are some tests that topicId is set two times.
I'm not sure if I'm not catching contexts.

Reply:
Sorry for this. It was a mistake. It is fixed now.

Reply:
It happens. 
NVM. 👍🏼